Added "Check PDF File for Password Protection" functionality

ronenPATeam opened the pull request.

Commit:
Added "Check PDF File for Password Protection" functionality

mateuszPATeam commented on PAteam.AcrobatReaderExtended.Library/Class1.cs:
```diff
@@ -7,6 +7,7 @@
 using System.IO;
 using System;
 using log4net;
+using System.Runtime.Remoting.Messaging;
```
Are you using this anywhere? If not, please remove

ronenPATeam replied:
Removed in the next PR.

mateuszPATeam commented on PAteam.AcrobatReaderExtended.Library/Class1.cs:
```diff
@@ -335,6 +336,65 @@ public static bool MergePdfFiles(DirectCollection<string> inputFilePaths, string
+                {
+                    _log.Debug("Direct.PDFExtended.Library - Checking pdf file: " + pdfFilePath + " for password protection");
+                }
+                if (pdfFilePath != null)
```
I would use string.IsNullOrEmpty to check if its empty

ronenPATeam replied:
Resolved in the next PR.

mateuszPATeam commented on PAteam.AcrobatReaderExtended.Library/Class1.cs:
```diff
@@ -335,6 +336,65 @@ public static bool MergePdfFiles(DirectCollection<string> inputFilePaths, string
+                }
+                if (pdfFilePath != null)
+                {
+                    if(pdfFilePath.EndsWith(".pdf"))
```
Maybe try to load file into File object and then check extension?

ronenPATeam replied:
Resolved in next PR, it will use the ValidateInput method that already being used in other places (just with an empty field name).

mateuszPATeam reviewed (changes requested):
Added comments to what I would change,

Commit:
-Changed file validation to use the ValidateInput method, with an empty field name. Because its already being used in other places and it checks everything. Tested and its working.
-Removed all the unused libraries (only one got removed System.Runtime.Remoting.Messaging).
-Added few more debug logger actions.
